src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -130,17 +130,17 @@
                 "UnnecessarySemicolonAfterTypeMemberDeclaration",
                 "UnnecessarySemicolonInTryWithResources",
                 "FinalLocalVariable",
-                "MissingSwitchDefault",
-                "UnnecessaryParentheses",
-                "MultipleStringLiterals",
-                "MultipleVariableDeclarations",
-                "AvoidDoubleBraceInitialization",
-                "ReturnCount",
-                "CovariantEquals",
-                "EqualsAvoidNull",
-                "HiddenField",
-                "NoClone",
-                "IllegalInstantiation"
+                "MissingSwitchDefault"
```

Comment:
please wrap this list to array to make it possible to use trailing comma, it will help to reduce diff

Reply:
You mean adding comman at the end of `MissingSwitchDefault`?

Reply:
I mean that at the moment, if you uncomment lines below, it wont compile. If you do something like
```
Collections.unmodifiableSet(new HashSet<>(Arrays.asList(new String[]{
"someCheck",
// "someCommentedCheck",
})));
```
and you uncomment second check, it will compile. This is what I am asking for

Reply:
It will give `Redundant array creation for calling varargs method `